spec/examples/metal/keyspace_spec.rb

```diff
@@ -104,6 +104,27 @@
     end
   end
 
+  describe "#client_compression" do
+    let(:client_compression) { :lz4 }
+    let(:client_mock) do
+      client_mock = mock
+      client_mock.expects(:connect).returns(true)
+      client_mock
+    end
+    let(:connect) do
+      Cequel.connect host: Cequel::SpecSupport::Helpers.host,
+          port: Cequel::SpecSupport::Helpers.port,
+          client_compression: client_compression
+    end
+    it "client compression settings get extracted correctly for sending to cluster" do
+      expect(connect.client_compression).to eq client_compression
+    end
+    it 'properly passes compression seetting to the driver' do
+      Cassandra.expects(:cluster).with(has_entries(compression: :lz4)).returns(client_mock)
+      expect(connect.client).to eq true
```

Comment:
I can't seem to find a getter for the settings passed to the driver. IMHO the mock's expectation here verifies the the cequel gem properly passes the setting to the driver. I'm open changing this especially since no other tests uses mocks. But without a getter I'm not exactly sure how to verify this.